Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Composable/Klarna/Managers/KlarnaTokenizationManager.swift

```diff
@@ -16,49 +16,30 @@ protocol KlarnaTokenizationManagerProtocol {
      
      - Returns: A `Promise<PrimerPaymentMethodTokenData>` which resolves to a `PrimerPaymentMethodTokenData` object on successful tokenization or rejects with an `Error` if the tokenization process fails.
      */
-    func tokenize(customerToken: Response.Body.Klarna.CustomerToken?, offSessionAuthorizationId: String?) -> Promise<PrimerCheckoutData>
+    func tokenizeHeadless(customerToken: Response.Body.Klarna.CustomerToken?, offSessionAuthorizationId: String?) -> Promise<PrimerCheckoutData>
+    func tokenizeDropIn(customerToken: Response.Body.Klarna.CustomerToken?, offSessionAuthorizationId: String?) -> Promise<PrimerPaymentMethodTokenData>
+    var mockedSuccessValue: Bool { get set }
```

Comment:
This mockSuccess value stands out to me a little. 
Is there a way to enable mocking this flow with a mocked object when needed? See `ThreeDsService` and `ThreeDSServiceProtocol` for an example.

Let me know if it would require a significant re-architecture, but it would be nice to isolate mocks

Reply:
Yes, i am not happy about this either, i will sort something out.